src/modules/BRepAlgoAPI.cxx

```diff
@@ -75,6 +79,34 @@ py::class_<BRepAlgoAPI_Algo, std::unique_ptr<BRepAlgoAPI_Algo, py::nodelete>, BR
 // cls_BRepAlgoAPI_Algo.def_static("operator delete_", (void (*)(void *, void *)) &BRepAlgoAPI_Algo::operator delete, "None", py::arg(""), py::arg(""));
 cls_BRepAlgoAPI_Algo.def("Shape", (const TopoDS_Shape & (BRepAlgoAPI_Algo::*)()) &BRepAlgoAPI_Algo::Shape, "None");
 
+/*
```

Comment:
it looks like it's all commented out and not exposed? whereas https://github.com/LaughlinResearch/pyOCCT/commit/4d49f20ec8672924f6f484c346e4cf9c5a51e591 wrote them as lambdas...i'm confused on what you're trying to do by commenting them out?